pkg/apis/workflow/v1alpha1/workflow_phase.go

```diff
@@ -4,17 +4,18 @@ package v1alpha1
 type WorkflowPhase string
 
 const (
-	WorkflowUnknown   WorkflowPhase = ""
-	WorkflowPending   WorkflowPhase = "Pending" // pending some set-up - rarely used
-	WorkflowRunning   WorkflowPhase = "Running" // any node has started; pods might not be running yet, the workflow maybe suspended too
-	WorkflowSucceeded WorkflowPhase = "Succeeded"
-	WorkflowFailed    WorkflowPhase = "Failed" // it maybe that the workflow was terminated
-	WorkflowError     WorkflowPhase = "Error"
+	WorkflowUnknown    WorkflowPhase = ""
+	WorkflowPending    WorkflowPhase = "Pending" // pending some set-up - rarely used
+	WorkflowRunning    WorkflowPhase = "Running" // any node has started; pods might not be running yet, the workflow maybe suspended too
+	WorkflowSucceeded  WorkflowPhase = "Succeeded"
+	WorkflowFailed     WorkflowPhase = "Failed" // it maybe that the workflow was terminated
```

Comment:
```suggestion
	WorkflowRunning    WorkflowPhase = "Running" // any node has started; pods might not be running yet, the workflow may be suspended too
	WorkflowSucceeded  WorkflowPhase = "Succeeded"
	WorkflowFailed     WorkflowPhase = "Failed" // it may be that the workflow was terminated
```

Reply:
(I know you didn't write this, but would be nice if we can fix it anyway?)